todo/src/scripts/storageController.js

```diff
@@ -0,0 +1,88 @@
+class StorageController {
+    constructor(name) {
+        this.init();
+        this.data;
+    }
+
+    init() {
+        if (!localStorage.getItem('toDo')){
+            localStorage.setItem('toDo', JSON.stringify(
+                {
+                    'currentTasks': [],
+                    'completeTasks': [],
+                }
+            ));
+        }
+        this.data = JSON.parse(localStorage.getItem('toDo'));
+        this.removeEmptyItems();
+    }
+
+    removeEmptyItems() {
+        this.data['currentTasks'] = this.data['currentTasks'].filter((item) => {return item != null});
+        this.data['completeTasks'] = this.data['completeTasks'].filter((item) => {return item != null});    
```

Comment:
`filter(item => item)` - does the same job

Reply:
```removeEmptyItems``` method was deleted.